### PR TITLE
S1t4 get emails

### PR DESCRIPTION
**Pull request by nico-dev-it**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,26 @@
 import './App.css';
+import {useEffect, useState} from 'react';
```
Comment (Harry-Johnston) [inline src/App.js:2]: remove the ;

File: src/Components/TestComponent/index.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const TestComponent = (props) => {
```
Comment (Harry-Johnston) [inline src/Components/TestComponent/index.js:1]: dont let this get merged into s1 - can you please remove

**Review by Harry-Johnston — CHANGES_REQUESTED** [review]
just some minor changes to make sure we dont forget test things in s1

File: src/App.js (hunk 1 above)
Comment (SlothSan) [reply src/App.js:2]: Removed all from the file.

File: src/App.js [hunk 3]
```diff
@@ -1,8 +1,26 @@
...
+        fetchAllEmailData();
+    }, [])
+
+    useEffect(() => {
+        console.log(allEmailSnippets)
+    }, [allEmailSnippets])
+
   return (
     <div className="App">
+      <TestComponent allEmailSnippets={allEmailSnippets} />
```
[the remark this answers is not in the document]
Comment (SlothSan) [reply src/App.js:23]: Removed, use this as reference when people need to add the data.

File: src/Components/TestComponent/index.js (hunk 2 above)
Comment (SlothSan) [reply src/Components/TestComponent/index.js:1]: Removed.

**Pull request merged**